java/code/src/com/redhat/rhn/domain/kickstart/KickstartInstallType.java

```diff
@@ -62,23 +69,14 @@ public boolean isRhel8OrGreater() {
      * @return if this installer type is rhel 7 or greater (for rhel8)
      */
     public boolean isRhel7OrGreater() {
-        return (isRhel6OrGreater() && !isRhel6());
+        return (!isRhel6());
     }
 
     /**
      * @return if this installer type is rhel 6 or greater (for rhel7)
      */
     public boolean isRhel6OrGreater() {
-        return (isRhel5OrGreater() && !isRhel5());
-    }
-
-    /**
-     * @return if this installer type is rhel 5 or greater (for rhel6)
-     */
-    public boolean isRhel5OrGreater() {
-        // we need to reverse logic here
-        return (!isRhel2() && !isRhel3() && !isRhel4() && !isFedora() && !isGeneric() &&
-                !isSUSE());
+        return !isFedora() && !isGeneric() && isSUSE();
```

Comment:
```suggestion
        return !isFedora() && !isGeneric() && !isSUSE();
```

Reply:
yes, found it too while debugging the unit tests failures